README.md

```diff
@@ -71,18 +71,18 @@ Config of the extension can be set in `excnfig.json` file.
 3. Add your network details in `hardhat.config.ts`.
 4. Deploy the factory using `INFURA_ID=<required> npx hardhat deploy --network <network>`.
 5. Edit the `factory_address` in `src/exconfig.ts`
-6. Edit the `bundler` url in `src/exconfig.ts` that points to your network and accepts requests for your EntryPoint.
+6. Set `bundler` in `src/exconfig.ts` to a bundler that points to your network and accepts requests for your EntryPoint.
 7. Run `yarn start`
 
 ### Local Network
 
 1. Run a local hardhat node with `npx hardhat node` or use the node inside the bundler repo.
-2. Deploy EntryPoint from [the infinitism repo](https://github.com/eth-infinitism/account-abstraction), you can find the instructions [below](#how-to-deploy-entrypoint-locally).
+2. Deploy EntryPoint from [the account-abstraction repo](https://github.com/eth-infinitism/account-abstraction), you can find the instructions [below](#how-to-deploy-entrypoint-locally).
 3. Edit the `entryPointAddress` in `src/exconfig.ts`.
 4. Deploy the factory using `npx hardhat deploy --network localhost`.
 5. Edit the `factory_address` in `src/exconfig.ts`
-6. Start a local bunder from [the infinitism repo](https://github.com/eth-infinitism/bundler), you can find the instructions [below](#how-to-run-bundler-locally).
-7. Edit the `bundler` to `http://localhost:9000/rpc` url in `src/exconfig.ts` that points to your network and accepts requests for your EntryPoint.
+6. Start a local bunder from [the bundler repo](https://github.com/eth-infinitism/bundler), you can find the instructions [below](#how-to-run-bundler-locally).
+7. Set `bundler` to `http://localhost:3000/rpc` in `src/exconfig.ts`.
```

Comment:
- Being more specific - there's more than one infinitism repo so 'the' infinitism repo is confusing.
- The bundler runs on port 3000 by default, and for the local set-up we have a specific value to use, and so shouldn't be suggesting there is some other constraint the user needs to satisfy (ie, that wording is appropriate for custom network but is confusing here)